docs/ide4j/vscode/README.md

```diff
@@ -0,0 +1,41 @@
+# Configuring VSCode for ICU4J
+
+To get ICU4J running in VSCode, including JUnit support:
+
+1. Install the Extension Pack for Java (`vscjava.vscode-java-pack`)
+2. In settings.json, remove the contents of `"java.project.resourceFilters"`. This will prevent the extension from dirtying your .project files in later steps:
+
+```
+    "java.project.resourceFilters": [
+    ]
+```
+
+3. Run `ant init` in the `icu4j` directory
+4. Run `sudo cp -R lib /external-libraries && chmod a+x /external-libraries`.
+    - *This is a big hack! It allows VSCode to find the required jar files.*
+    - The same outcome can also be achieved by modifying the paths in the `.classpath` files, as described below.
```

Comment:
suggestion: tell people to edit `.classpath` files rather than go through steps 4.

Reply:
Done.